NEWS

```diff
@@ -26,6 +26,10 @@ Several important changes and enancements were added:
      cat /proc/self/maps | grep '\[vvar]'
 * Support for glibc version 2.21 added.  To see if this affects you, do:
      ls -l /lib*/libc.so.6 /lib/*/libc.so.6
+* The environment variable DMTCP_GDB_ATTACH_ON_RESTART was added.  Setting
```

Comment:
Not sure if we should be exposing internal debugging env vars to the end user. I haven't seen any other release notes from other projects following this suit.